### ISSUE TITLE
Create web form for "Events"

### ISSUE THREAD

**Issue by bnvk** [opensourcedesign/opensourcedesign.github.io#81]
As per the goal of making it easy for designers who are not comfortable using GitHub to participate in OSD, we should make a nice web form so that people can "Add An Event" without having to create a Github account.

**Comment by grahamperrin** [comment]
Can this be made CalDAV-friendly from the outset?

**Comment by elioqoshi** [comment]
Yeah, CalDav is important here I believe.

**Comment by simonv3** [comment]
Yeah, that'd be great. Do you know what the specs for that are? Can a webpage be caldav friendly on its own? Or do you need to create an ics?

**Comment by Erioldoesdesign** [comment]
In the interests of things being 'friendly' adding what I assume is what CalDav is with a link ;) https://caldavsynchronizer.org/

**Comment by jdittrich** [comment]
Since
* Forms need some background magic to work on static sites
* This has not been touched for several years

I suggest closing this.
